build-system/tasks/compile.js

```diff
@@ -148,7 +148,8 @@ function compile(entryModuleFilename, outputDir,
         only_closure_dependencies: true,
         output_wrapper: wrapper,
         create_source_map: intermediateFilename + '.map',
-        source_map_location_mapping: '|http://localhost:8000/',
+        source_map_location_mapping:
+            '|https://raw.githubusercontent.com/ampproject/amphtml/master/',
```

Comment:
so if we had the `tag` should be something like https://rawgit.com/ampproject/amphtml/1454976295735/src/

ex: https://rawgit.com/ampproject/amphtml/1454976295735/src/3p.js